dff/core/engine/conditions.py

```diff
@@ -128,12 +129,13 @@ def any_condition_handler(ctx: Context, actor: Actor, *args, **kwargs) -> bool:
 
 @validate_arguments
 def all(cond_seq: list, *args, **kwargs) -> Callable:
```

Comment:
Should we add Callable signature (☝️): `Callable[[Context, Actor, Any, Any], bool]` (AFAIK) here.